spec/javascript/support/puppeteer/steps/leaseUpStatusSteps.js

```diff
@@ -0,0 +1,59 @@
+import sharedSteps from './sharedSteps'
+
+export const openStatusModalSelector = '.form-modal_form_wrapper'
```

Comment:
this ends up being a lot of selectors. Normally with strings I'd use CAPITAL_SNAKE_CASE but so far we haven't used that for selectors and there are SO MANY SELECTORS.

I am also inconsistent about using Selector in the variable honestly just because some of the variable names end up getting too long. 

If we can agree on some conventions I'd be happy to update though.